test/verify/run-tests

```diff
@@ -1,18 +1,91 @@
 #!/usr/bin/python3
 
 import glob
-import imp
 import os
 import string
 import subprocess
 import sys
 import unittest
+import dataclasses
+import errno
+import fcntl
+import time
+import socket
+
+import importlib.machinery
+import importlib.util
 
 import parent
 import testlib
 import testvm
 
 sys.dont_write_bytecode = True
+os.environ['PYTHONUNBUFFERED'] = '1'
+
+
+@dataclasses.dataclass
+class Test:
+    test_id: int
+    command: list
+    process: subprocess.Popen = None
+    retries: int = 0
+    output: bytes = b""
+
+
+def print_tap(test, print_ok=True):
```

Comment:
Sorry for ambiguity earlier.. `print_test()` was quite fine. I meant to replace `print_ok` with `print_tap`, as it's not always "ok" :grin: